README.md

```diff
@@ -1,14 +1,22 @@
 shavar-list-creation
 ====================
-    usage: lists2safebrowsing.py
+This script fetches blocklist `.json` from urls (such as
+[shavar-prod-lists](https://github.com/mozilla-services/shavar-prod-lists)) and
+generates safebrowsing-compatible digest list files to be served by
+[shavar](https://github.com/mozilla-services/shavar).
 
-A shavar.ini file must exist in the same directory that specifies the following parameters. See sample_shavar_list_creation.ini for examples.
+# Requirements
+A `shavar_list_creation.ini` file must exist in the same directory that specifies the following parameters. (See `sample_shavar_list_creation.ini` for an example.)
 
-Commits to shavar-list-exceptions trigger rebuilds of shavar-list-creation like so:
+# Run
+```
+./lists2safebrowsing.py
+```
 
-1. A commit is made to https://github.com/mozilla-services/shavar-list-exceptions
-2. That commit triggers a Jenkins workflow that will pull down the current exception list
-3. Jenkins then pulls down the master codebase from https://github.com/mozilla-services/shavar-list-creation and builds the list
-4. The completed list is placed in the s3 bucket for production
+# Usage
+This is run by [a Jenkins deployment job every 30 minutes](https://deploy.mozaws.net/view/Shavar%20Stage/job/Shavar%20List%20Creation%20Stage/). The job:
```

Comment:
Can we please remove this? That server isn't intended for public use/viewing. No real point in having it here anyway.